app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -156,10 +158,56 @@ private void initView() {
         adapter = new SessionsAdapter(getContext());
         binding.recyclerView.setAdapter(adapter);
 
+        GestureDetector gestureDetector = new GestureDetector(getContext(), new GestureDetector.OnGestureListener() {
```

Comment:
Could you please create new class or change the location to initialize? because it seems `initView` became so big and many methods of GestureDetector are not used. 

In addition, `ignoreMotionEventTillDown` should be changed to `ignoreMotionEventOnScroll`. It's just typo.
Sorry for splitting reviews. 🙇